samples/language/MultiFileProject/qsharp.json

```diff
@@ -1 +1,16 @@
-{}
+{
+    "lints": [
+        {
+            "lint": "deprecatedFunctionConstructor",
```

Comment:
I thought we weren't going through with this change?

Reply:
I was actually thinking of removing this lint altogether since we are going to support both for the near future.

Reply:
Good point -- I left it in just so we'd get warnings, but I'll remove that for now, now that we've decided that lint will go away.